### ISSUE TITLE
keybase/macOS - Command Q Does not Quit / Adhere to Apple's Human Interface Guidelines

### ISSUE THREAD

**Issue by colinbjohnson** [keybase/client#20580]
According to Apple's Human Interface Guidelines, pressing "Command-Q" should "Quit the app" - these guidelines are defined here: https://developer.apple.com/design/human-interface-guidelines/macos/user-interaction/keyboard/.

Within Keybase, the Human Interface Guidelines are not adhered to, and rather, "Command-Q" reads "Minimize to Tray".

This behavior is not in accordance with the Human Interface Guidelines standards.

I have attached a screenshot of the functionality that does no adhere to these standards.

![Keybase - Quit Does Not Adhere to Human Interface Guidelines](https://user-images.githubusercontent.com/1594843/67516022-b53efa80-f654-11e9-9a04-b1d67aac0517.png)

**Comment by revonoc** [comment]
Agreed - there should just be a settings option to have command-Q quit instead. If you haven't figured it out, for now you can quit by clicking on the Keybase logo in your menubar up top and clicking quit

**Comment by sgoodwin** [comment]
This is an easy thing to fix, I'm happy to make the PR for it if we can do this. I looked through the discussion and it seems there was some intent to make things the way it is on purpose?

It's a pretty suspicious experience to have a Mac app that doesn't quit with command-Q. Also suspicious when an app keeps inserting itself into the "start on launch" list without asking.

**Comment by mgibson323** [comment]
I am seeing this issue as well. Cmd+Q does not quit the app, and I do not have a “quit” option from the Keybase menu on the menu bar. I have to force quit the app to quit it.
The only other times I’ve seen this issue with software have been with spyware, malware, and viruses. I’m not implying Keybase is these, but it shouldn’t behave like them, either.

**Comment by ragunaru** [comment]
This is not how a native macOS app should behave. No option in the preferences to disable this forced behavior and also to remove it from "start on launch". This will get a hard uninstall from me until these issues are fixed.

**Comment by gregsadetsky** [comment]
Binding the "Minimize to Tray" to Command-Q is indeed very strange...!

It's not a perfect solution, but as a temporary fix, you can do this to revert Command-Q to mean Quit:

Go to System Preferences -> Keyboard -> Shortcuts -> App Shortcuts

Add a new shortcut using the "+" button.

Select Keybase.app in the list of apps, then in the "Menu Title" input type (exactly) "Quit Keybase Completely", then focus the "Keyboard Shortcut" input, and type Command-Q. Click the "Add" button to complete.

Then add another shortcut using the "+" button, select Keybase.app again, type "Minimize to Tray" in the "Menu Title" input, and use any other shortcut, for example, Command-Option-Q, the click the "Add" button.

This is what adding the first shortcut looks like:

![Screen Shot 2021-05-30 at 2 10 04 PM](https://user-images.githubusercontent.com/1017304/120115355-4e979200-c151-11eb-9cbd-cc915eb5988f.png)
